File: legacy/gateway-py/gateway/app/routers/health.py
from fastapi import APIRouter
from gateway.app.config import settings
router = APIRouter()
@router.get("/healthz")
async def healthz():
    return {"status": "ok", "service": settings.SERVICE_NAME, "version": "0.1.0"}
@router.get("/.well-known/jwks.json")
async def jwks():
    return {"keys": []}  # populated on first key use

File: gateway/src/health.rs
//! Health + JWKS endpoints.

use axum::{extract::State, Json};
use serde_json::{json, Value};

use crate::state::AppState;

pub async fn healthz(State(state): State<AppState>) -> Json<Value> {
    Json(json!({
        "status": "ok",
        "service": state.config.service_name,
        "version": env!("CARGO_PKG_VERSION"),
    }))
}
